mypy/errors.py

```diff
@@ -618,8 +618,12 @@ def report_internal_error(err: Exception,
 
     # Print "INTERNAL ERROR" message.
     print('{}error: INTERNAL ERROR --'.format(prefix),
-          'please report a bug at https://github.com/python/mypy/issues',
-          'version: {}'.format(mypy_version),
+          'Please try using master\n'
+          'https://mypy.rtfd.io/en/latest/common_issues.html#using-pre-release-mypy',
+          file=stderr)
+    print('If this crash persists, please report a bug at https://github.com/python/mypy/issues',
```

Comment:
I'd like to hear about all crashes with a traceback, even if they are not reproducible. Sometimes a traceback is sufficient to figure out what went wrong. What about reverting to the original message if we are displaying a traceback?